universal/src/formats/json/impl/accept.hpp

```diff
@@ -95,11 +100,13 @@ bool WriteEnd(Handler& handler, const Cell& cell) {
   using ArrayIterator = typename Cell::ArrayIterator;
 
   return cell.Visit(
-      [&handler](const SubRange<MemberIterator>&) mutable -> bool {
-        return handler.EndObject();
+      [&handler, size = cell.GetSize()](
+          const SubRange<MemberIterator>&) mutable -> bool {
+        return handler.EndObject(size);
```

Comment:
Let's compute `size` on-demand right here using the `SubRange` argument. It should save up some space in the Stack and give a marginal performance improvement

Reply:
at this point std::distance(sub_range.begin, sub_range.end) == 0
as I understood StackCell::sub_range_.begin moved to StackCell::sub_range_.end by StackCell::Next

Reply:
By the time of the call WriteEnd for array, cell.sub_range_.begin is equal to cell.sub_range_.end.
Therefore we cannot calculate size of array here nor get value's parent for getting initial sub_range_.begin.
Please tell me the direction of finding a solution.

Reply:
Ah, I'm sorry, you are right.
It's unfortunate that rapidjson requires this size.